### ISSUE TITLE
Extension crashed

### ISSUE THREAD

**Issue by greenlaw110** [rmassidda/ulauncher-firefox-history#7]
> The extension crashed. Ensure that you read and followed the instructions on the extension [Github](file2:///usr/share/ulauncher/preferences/dist/index.html) page, and check the error log and report the error otherwise.

![image](https://github.com/rmassidda/ulauncher-firefox-history/assets/216930/8b4f9ced-a5fd-4876-96ed-36a35052697f)

**Comment by rmassidda** [comment]
Hey! Thanks for the feedback. Could you provide additional info? There should be a log attached to the crash.

**Comment by greenlaw110** [comment]
Hey @rmassidda where can I get the log?

**Comment by kskollepara** [comment]
[last.log](https://github.com/rmassidda/ulauncher-firefox-history/files/14165134/last.log) 
I have the same problem. If you are looking for `last.log` of ulaucher, I attach it. If its another log file, please let me know the path (on Ubuntu 22)

**Comment by rmassidda** [comment]
I haven't been able to replicate the issue on my system, however I just pushed a new version which should include more info in the logs. @kskollepara, if you could provide the `last.log` file after upgrading the extension it would be extremely useful.

**Comment by kskollepara** [comment]
Thank you for your reply @rmassidda . Here is the new log file 
[last.log](https://github.com/rmassidda/ulauncher-firefox-history/files/14482311/last.log)

**Comment by rmassidda** [comment]
Thanks! I'll try to replicate it on a virtual machine with Ubuntu 22.04.
Which version of Firefox are you using? Is it installed through snap?

**Comment by kskollepara** [comment]
Yes, it is installed via snap. Currently, the Firefox version is 123.0. However, I believe the problem appeared with 122.x as well

**Comment by kskollepara** [comment]
Hello @rmassidda , do you have any news about the problem?

**Comment by rmassidda** [comment]
Hey @kskollepara, I wasn't able to replicate the issue on the virtual machine. Also, despite the additional debug messages I added in the code, there are no hints in the `last.log` on what could have caused the termination error.

I have not used Ulauncher in a long time, but feel free to submit a pull request. :)

Sorry for not being able to provide more help!
